.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -152,32 +158,34 @@ run_tests() {
   (
     set -e
     echo Using PR container image: ${TAG_NAME}
-    yarn test
+    yarn "$project"
   ) |& tee "/tmp/${LOGFILE}"
 
   RESULT=${PIPESTATUS[0]}
 
   pkill Xvfb
 
-  mkdir ${ARTIFACT_DIR}/test-results
-  cp -a /tmp/backstage-showcase/e2e-tests/test-results/* ${ARTIFACT_DIR}/test-results
+  mkdir -p ${ARTIFACT_DIR}/$project/test-results
+  cp -a /tmp/backstage-showcase/e2e-tests/test-results/* ${ARTIFACT_DIR}/$project/test-results
 
   ansi2html <"/tmp/${LOGFILE}" >"/tmp/${LOGFILE}.html"
   cp -a "/tmp/${LOGFILE}.html" ${ARTIFACT_DIR}
```

Comment:
showcase log file will be overwritten by showcase-rbac test runs. better to move it inside the $project like `${ARTIFACT_DIR}/${project}`

Reply:
sorry for my miss understanding, but why will be overwritten if I'm already using `${ARTIFACT_DIR}/$projec` ?

Reply:
As per my understanding you are storing the `test-results` as per the project but we also have [test-log.html](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/janus-idp_backstage-showcase/1200/pull-ci-janus-idp-backstage-showcase-main-e2e-tests/1783100244364365824/artifacts/e2e-tests/janus-idp-backstage-showcase/artifacts/test-log.html). which is currently outside the `$project` and both `showcase` and `showcase-rbac` will have there own `test-log.html` which needs to be moved inside the `$project`. like: 

```suggestion
  cp -a "/tmp/${LOGFILE}.html" ${ARTIFACT_DIR}/${project}
```